src/ShapeCrawler/Drawing/CroppingFrame.cs

```diff
@@ -1,36 +1,22 @@
 using System;
-using DocumentFormat.OpenXml;
 using ShapeCrawler.Exceptions;
-using A = DocumentFormat.OpenXml.Drawing;
 
 namespace ShapeCrawler.Drawing;
 
 /// <summary>
 ///     The portion within an image which is visible. 
 /// </summary>
 /// <remarks>
-///     Values are a fraction of the whole image, e.g. `0.25` means that 25% of the image is removed
+///     Values are a percentage of the whole image, e.g. `25` means that 25% of the image is removed
 ///     on the given edge.
 ///     An image which is fully visible will have 0 for all values.
 /// </remarks>
 /// <param name="left">Portion of image along left edge of source picture which will not be displayed.</param>
 /// <param name="right">Portion of image along right edge of source picture source picture which will not be displayed.</param>
 /// <param name="top">Portion of image from top edge of source picture source picture which will not be displayed.</param>
 /// <param name="bottom">Portion of image from bottom edge of source picture source picture which will not be displayed.</param>
-public record CroppingFrame(decimal left, decimal right, decimal top, decimal bottom)
+public readonly record struct CroppingFrame(decimal left, decimal right, decimal top, decimal bottom)
```

Comment:
My IDE complains about naming `left`, `right`, `top` and `bottom`

> Name 'left' does not match rule 'non_field_members_should_be_pascal_case'. Suggested name is 'Left'.

and I agree with him🙂

Could you please capitalize this.

I would also adjust their summary to mention units of measurement "Portion of image _in percentages_ along left edge of the source picture, which will not be displayed."

Reply:
Yah, these are actually properties so should be Pascal Case. Usually these would be primary constructor parameters, so camelCase

2dc11630e6b6a643847fdf3f2077d1940ff6da8d